app/reducers/index.jsx

```diff
@@ -1,7 +1,9 @@
 import { combineReducers } from 'redux'
 
 const rootReducer = combineReducers({
-  auth: require('./auth').default,  
+  auth: require('./auth').default,
+  products: require('./products').default,
+  product: require('./product').default,
 })
 
```

Comment:
default is how they did it in Bones- don't know what that actually does but it seems to work